vite.config.ts

```diff
@@ -93,6 +93,7 @@ const getConfigJson = async (url: string, config: UserConfig) => {
 }
 
 export default defineConfig(async ({ mode, command }) => {
+  const watch = process.argv.includes('--watch')
```

Comment:
Please do it like here: 

https://github.com/owncloud/web/blob/c037d2ec0510ef4dc9fee7c24f4c25aff04f2f95/vite.config.ts#L118